packages/@aws-cdk/aws-kms/lib/key.ts

```diff
@@ -1,4 +1,4 @@
-import { Construct, DeletionPolicy, Output, PolicyDocument, PolicyStatement } from '@aws-cdk/cdk';
+import { Construct, DeletionPolicy, Output, PolicyDocument, PolicyStatement, resolve } from '@aws-cdk/cdk';
```

Comment:
At this point I would `import cdk = require('@aws-cdk/cdk');`.

Reply:
✅

Reply:
You'll forgive me - I forgot to push this change and merged. We'll clean this up later.